core/client/app/templates/setup/two.hbs

```diff
@@ -18,22 +18,24 @@
             </div>
             <a class="edit-account-image" href="#"><i class="icon-photos"><span class="sr-only">Upload an image</span></i></a>
         </figure>
-        <div class="form-group">
+        {{#gh-form-group errors=errors prop="email"}}
             <label for="email-address">Email address</label>
             <span class="input-icon icon-mail">
-                {{input type="email" name="email" placeholder="Eg. john@example.com" class="gh-input" autofocus="autofocus" autocorrect="off" value=email}}
+                {{gh-input type="email" name="email" placeholder="Eg. john@example.com" class="gh-input" autofocus="autofocus" autocorrect="off" value=email model=this prop="email"}}
```

Comment:
I have to say I'm not a fan of this at all.  Why is a text input being given an entire Controller instance so it can validate an input is an email address?